nix/yazi-unwrapped.nix

```diff
@@ -0,0 +1,52 @@
+{ makeRustPlatform
+, rustToolchain
+, version ? "git"
+, lib
+
+, installShellFiles
+, stdenv
+, darwin
+
+, imagemagick
+}:
+
+(makeRustPlatform { cargo = rustToolchain; rustc = rustToolchain; }).buildRustPackage rec {
+  pname = "yazi";
+  inherit version;
+
+  src = ../.;
+
+  cargoLock.lockFile = ../Cargo.lock;
+  cargoBuildFlags = [ "-p" "yazi-fm" "-p" "yazi-cli" ];
```

Comment:
Maybe add `yazi-cli` to [`default-members`](https://github.com/sxyazi/yazi/blob/c65a14c0e650c92cce52fe507b9f84a873cae44d/Cargo.toml#L4) instead of additional build flags?